joomla/joomla-cms
# [Fix #13083]Text Filters text shown on Permissions tab in Global Configuration

Pull Request for Issue #13083 

### Summary of Changes

reset the description value

### Testing Instructions

Go to Global Configuration.  Click on Permissions tab.
see:
![screenshot from 2016-12-04 14-20-02](https://cloud.githubusercontent.com/assets/305153/20866804/c5e0a7dc-ba2d-11e6-94f6-84797051a857.png)

Apply the patch
make sure the extra text is gone now.

### Documentation Changes Required

none
